requirements.txt

```diff
@@ -5,8 +5,10 @@ celery==4.0.2
 redis==2.10.5
 giturl.py==0.2.1
 GitPython==2.1.5
-scancode-toolkit==2.0.1
+scancode-toolkit==2.2.1
+requests==2.9.1
 
 # for testing
 pytest
-pytest-django
\ No newline at end of file
+
+# pytest-django
```

Comment:
Why do you remove this requirement?

Reply:
This was giving some error!!

Reply:
Which error? We do not remove dependencies without a good reason.

Reply:
Earlier when I created virtual env via pycharm it gave an error. But when i did that directly from terminal it works fine. So, I guess that's my mistake. Sorry, for that.